src/Views/Admin/Pages/templates/setup-step-template.php

```diff
@@ -0,0 +1,9 @@
+<div class="section-card">
+	<div class="section-card--label">
+		<span class="dashicons dashicons-<?php echo $dashicon; ?>"></span>
+		<?php echo $labelText; ?>
+	</div>
+	<div class="section-card--action">
+		<a class="button button-primary" href="<?php echo $actionLocation ?? '#'; ?>"><?php echo $actionText; ?></a>
```

Comment:
@kjohnson I donot think this operator support by 5.6 PHP. It will throw a fatal error. Can you confirm?

Reply:
The Null Coalescing Operator was introduced in PHP7. I'll update to use an `isset()` check for PHP5.2 compatibility.